### PR TITLE
Test fixes

### PR DESCRIPTION
**Pull request by minnerthecat**
Fixed tests to be less destructive to database and not assume that data will be there.
Removed HomeController and EventController tests, they aren't working properly.

### CODE REVIEW

**Comment by minnerthecat** [comment]
Don't merge this yet!

**Comment by minnerthecat** [comment]
Ok good to go!

File: partEZ/tests/databasemocks/HomeControllerTest.php [hunk 1]
```diff
@@ -1,35 +0,0 @@
...
-    public function testGetUsersEvents()
-    {
-
-        // $this->mock->shouldReceive('getByUID')->once()->andReturn(1);
-        // $this->app->instance('\App\Event', $this->mock);
-        // $response = $this->controller->getUsersEvents();
-        // $this->assertEquals(1, $response->eid);
-
-    }*/
-}
```
Comment (bmansell) [inline partEZ/tests/databasemocks/HomeControllerTest.php:35]: Are we going to rewrite or need different versions of these tests later? If not then this PR looks good.

Comment (minnerthecat) [inline partEZ/tests/databasemocks/HomeControllerTest.php:35]: We probably should

**Comment by delroyhiebert** [comment]
looks good to me, merging in

**Pull request merged**